mount/mount.go

```diff
@@ -66,8 +68,5 @@ func WithTempMount(mounts []Mount, f func(root string) error) (err error) {
 		return errors.Wrapf(uerr, "failed to mount %s", root)
 	}
 
-	if uerr = f(root); uerr != nil {
-		return errors.Wrapf(uerr, "failed to f(%s)", root)
-	}
-	return nil
+	return f(root)
```

Comment:
Why the remove? I think it'll make it easier to see where the error from the callback starts, the if is not needed, but I think the Wrap adds value

Reply:
I don't think it is valuable since normally the closure and the caller will wrap and this just ends up adding a line if you do stack output. That is a very subjective viewpoint though so I added back.

Reply:
I see your point, I was just assuming the closure wouldn't do it correctly. Thanks for adding it back, would have accepted without after your comment though :innocent: